pkg/services/meta/meta.go

```diff
@@ -25,7 +25,7 @@ const (
 
 	// rootKey is the key for the last known state root in KV data base
 	// associated with MPT.
-	rootKey = 0x00
+	rootKey = 0xff
```

Comment:
back compatible change?

Reply:
Shouldn't be a problem, it's an experimental feature, not available in production anywhere.